base/acme/src/main/java/org/dogtagpki/acme/issuer/PKIIssuer.java

```diff
@@ -123,6 +123,7 @@ public String issueCertificate(String csr) throws Exception {
         AuthorityID aid = null;
         X500Name adn = null;
 
+        caClient.login();
```

Comment:
Did you get any problem if you didn't call the `login()` explicitly? Normally the authentication should happen automatically when the client executes a REST operation that requires authentication, so the `login()` isn't really required. It was originally added as a workaround for another issue, but now it's mainly used to get the `AccountInfo` or to force an early authentication. It's harmless to leave it here, but I was just wondering if this somehow becomes required.

Reply:
Yes, there were problems.  Session pre-establishment seems to be pequired for *profile* authentication, which is not handled by the usual servlet interceptors.  Perhaps it could be changed, I haven't investigated deeply because the explicit login resolved it.

Reply:
Yeah, as discussed on IRC, the profile service is a legacy service that uses a non-standard authentication mechanism, so it makes sense if it requires an explicit `login()`. Hopefully in the future we can rewrite it into a proper REST service. Thanks!